jenkins-job-builder/builders.yml

```diff
@@ -144,9 +144,9 @@
     name: copy-pw-files
     builders:
         - shell: |
-            mkdir -p $HOME/.ipa/alias
+            rm -rvf $HOME/.ipa/alias
             echo $FREEIPACI_PASSWORD > $HOME/.ipa/.dmpw
```

Comment:
The ~/.ipa directory might not exist at this point. Please delete the whole .ipa directory and recreate it again before writing in it.